app/assets/javascripts/constants/style_constants.es6.jsx

```diff
@@ -88,6 +88,10 @@
             display: 'flex',
             marginBottom: '18px',
           },
+          description: {
+            lineHeight: '15px',
+            fontWeight: '200',
```

Comment:
Did you add a new `fontWeight`? `fontWeight` should be above `lineHeight` as well.